app/src/main/kotlin/cz/covid19cz/erouska/jobs/AutoRestartJob.kt

```diff
@@ -0,0 +1,84 @@
+package cz.covid19cz.erouska.jobs
+
+import android.app.AlarmManager
+import android.app.PendingIntent
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Handler
+import cz.covid19cz.erouska.BuildConfig
+import cz.covid19cz.erouska.service.CovidService
+import cz.covid19cz.erouska.utils.L
+import org.threeten.bp.ZonedDateTime
+
+class AutoRestartJob : BroadcastReceiver() {
+
+    companion object {
+        const val EXTRAKEY_START = "AUTO_RESTART_START"
+        const val EXTRAKEY_CANCEL = "AUTO_RESTART_CANCEL"
+
+        private const val ACTION = "EROUSKA_ALARM"
+
+        private val SCHEME = "erouska" + if (BuildConfig.FLAVOR == "dev") "-dev" else ""
+        private val URI = Uri.parse("$SCHEME://auto-restart")
+
+        const val INTERVAL = 24 * 60 * 60 * 1000L // 24 hours
+    }
+
+    override fun onReceive(context: Context?, intent: Intent?) {
```

Comment:
Nejde tady do nullable dat pryc?

Reply:
Jde. To jsou nějaký divný manýry teda, že si můžu vybrat, jestli to bude nullable nebo ne :-D AS mi tam to nullable vyplnilo samo...

Reply:
Uvidime v crashlytics..Ja si myslim ,ze proste jen ty framework tridy nejsou spravne oanotovany, takze kotlin nevi, ze to je nonnull